Dockerfile

```diff
@@ -1,19 +1,18 @@
-FROM quay.io/ukhomeofficedigital/openjdk11:v11.0.5_10
+FROM quay.io/ukhomeofficedigital/alpine:v3.13
 
 ENV USER user_hocs_workflow
-ENV USER_ID 1000
 ENV GROUP group_hocs_workflow
 ENV NAME hocs-workflow
 ENV JAR_PATH build/libs
 
-RUN yum update -y glibc && \
-    yum update -y nss && \
-    yum update -y bind-license
+USER root
```

Comment:
We have to swap to root here as the image is set up to make us a user without permission to install, see https://quay.io/repository/ukhomeofficedigital/alpine/manifest/sha256:81f32a5fca1f3f053703c0c2488fbd6b6bed4f5af8108a2ae186095d34f8c9e6 for more info on the alpine image